glide.lock

```diff
@@ -1808,7 +1808,7 @@ imports:
   - pkg/util/proto
   - pkg/util/proto/validation
 - name: k8s.io/kubernetes
-  version: d1702ec28ea20dfa3111587b45875232702414ca
+  version: 48e4040dfae5691fe877037c75ac7c8c061260a7
```

Comment:
@deads2k this bumps because of publisher-bot doing its job?

Reply:
> @deads2k this bumps because of publisher-bot doing its job?

yeah